pom.xml

```diff
@@ -67,7 +67,7 @@ Note: The maven build currently needs to be started from the javaslang root dir
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <assertj.core.version>3.5.2</assertj.core.version>
         <eclipse.lifecycle.mapping.version>1.0.0</eclipse.lifecycle.mapping.version>
-        <java.version>1.8</java.version>
+        <java.version>1.9</java.version>
```

Comment:
This will compile Java 9 binaries, I think it should stay 1.8. Only the JDK used to compile has to change, right?